h/assets.yaml

```diff
@@ -214,6 +214,26 @@ inject_css:
         - h:static/styles/variables.scss
         - h:static/styles/mixins/icons.scss
 
+# the H website
+site_js:
+  output: scripts/site-bundle.js
```

Comment:
I left it out as the bundle is very small at this stage so I didn't see much point. Happy to change that for consistency if you'd prefer.